tasks/omnibus.py

```diff
@@ -379,3 +379,26 @@ def manifest(
         omnibus_s3_cache=False,
         log_level=log_level,
     )
+
+@task
+def rpath_edit(ctx, install_path, target_rpath_dd_folder, macos=False):
+    for file in glob.iglob(f"{install_path}/**/*", recursive=True):
+        ext = os.path.splitext(file)[1]
+        if (ext != "" and "so" not in ext and "dylib" not in ext) or "json" in ext or not os.path.isfile(file):
```

Comment:
Would it be worth to use something like `ctx.run(f"file -b --mime-type {file}")` instead? I'm afraid of missing some files that also need to be patched which for some reason can't be identified just by extension.

Reply:
We're using the extensions to skip some files we know we want to remove, but can be considered depending we manage to reduce patching time.